examples/js/libs/inflate.min.js.map

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Well its meant for the inflate.min.js, If I dont include it, it gives an error saying that, `inflate.min.js.map not found`. I have taken that from the [Zlib.js](https://github.com/imaya/zlib.js/tree/master/bin) repository.